### ISSUE TITLE
Getting Error: not implemented

### ISSUE THREAD

**Issue by mhaagens** [divmain/rapscallion#6]
Great library! I'm trying to use this with Express, but can't seem to get it to work.
I'm getting this error;

```
Error: not implemented
    at Readable._read (_stream_readable.js:470:22)
    at Readable.read (_stream_readable.js:348:10)
    at resume_ (_stream_readable.js:737:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

This is my render function;

```
const renderApp = (req, res) => {
    const context = {};

    const application = renderToStream(
        <StaticRouter location={req.url} context={context}>
            <App />
        </StaticRouter>
    );

    const html = streamTemplate`<!doctype html><html>
      <head>
        <title>Test</title>
      </head>
      <body>
        <div id="root">${application}</div>
        ${process.env.NODE_ENV === 'production' ? '<script type="text/javascript" src="/assets/vendor.js"></script>' : ''}
        <script src="/assets/app.js"></script>
      </body>
    </html>`

    if (context.url) {
        res.writeHead(302, { Location: context.url });
        res.end();
    } else {
        toNodeStream(html).pipe(res)
    }

};
```

Any idea as to what's going wrong here?

**Comment by divmain** [comment]
You're the first person to pull this down and give it a try! Let me get back to you tomorrow morning, it has been a couple of late nights to put my thoughts into code.

**Comment by mhaagens** [comment]
I'm really excited for this library, great work!
I'm using renderToString from rapscallion now and it shaved off roughly ~200ms from my server render, so I'm very excited about getting the streaming working. Whenever you have time to take a look at it that would be awesome :)

**Comment by mhaagens** [comment]
I'll just post this here as well, when used with React Router renderToString returns;
```
TypeError: Cannot read property 'createHref' of undefined
```
in case you're using this with React Router as well and know what's causing this?

**Comment by divmain** [comment]
@mhaagens, the stream issue should be fixed and release in `1.0.2`.  Could you open an issue regarding React Router, possibly with a minimal reproduction?  I haven't done too much testing yet, so real-world examples will be hugely useful to me.

Bed now though, for realz this time.

**Comment by mhaagens** [comment]
Thanks for the super fast fix! I'll open an issue regarding React Router.
And yeah, get some sleep :+1:
